san/batch.py

```diff
@@ -11,24 +12,36 @@ def __init__(self):
     def get(self, dataset, **kwargs):
         self.queries.append([dataset, kwargs])
 
-    def execute(self):
-        result = []
+    def __create_batched_query_string(self):
         batched_queries = []
 
-        for idx, q in enumerate(self.queries):
-            batched_queries.append(get_gql_query(idx, q[0], **q[1]))
-
-        gql_string = self.__batch_gql_queries(batched_queries)
-        res = execute_gql(gql_string)
+        for idx, query in enumerate(self.queries):
+            [metric, slug] = query[0].split('/')
+            if metric in V2_METRIC_QUERIES:
+                batched_queries.append(
+                    san.sanbase_graphql.get_metric(
+                        idx, metric, slug, **query[1]))
+            else:
+                batched_queries.append(get_gql_query(idx, query[0], **query[1]))
+        self.__batch_gql_queries(batched_queries)
+        return self.__batch_gql_queries(batched_queries)
+
+    def __transform_batch_result(self, executed_gql):
```

Comment:
`executed_gql` -> `graphql_result` :?

Reply:
It does sound better, change is on the way